src/components/menu/menu-test.stories.tsx

```diff
@@ -1,13 +1,26 @@
 import React, { useState } from "react";
 import { action } from "@storybook/addon-actions";
-
-import { Menu } from ".";
+import {
+  Menu,
+  MenuDivider,
+  MenuSegmentTitle,
+  MenuTitleProps,
+  ScrollableBlock,
+} from ".";
 import { MenuType } from "./menu.context";
-import MenuItem from "./menu-item";
+import MenuItem, { MenuWithChildren } from "./menu-item";
 import MenuFullscreen, { MenuFullscreenProps } from "./menu-full-screen";
 import Search from "../search";
 import NavigationBar, { NavigationBarProps } from "../navigation-bar";
 import GlobalHeader from "../global-header";
+import Box from "../box/box.component";
+import Typography from "../typography/typography.component";
+import useMediaQuery from "../../../src/hooks/useMediaQuery";
+import { MenuProps } from "./menu.component";
+import { MenuDividerProps } from "./menu-divider";
+import { ScrollableBlockProps } from "./scrollable-block";
+
+const menuTypes = ["white", "light", "dark", "black"] as const;
```

Comment:
Instead of defining this as a `const` locally, use your `MenuType` import from `Menu.context`

Reply:
Something like this should work `const menuTypes: MenuType[] = ["white", "light", "dark", "black"];`